docs/DeveloperGuide.adoc

```diff
@@ -721,20 +721,67 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |user |add multiple phone numbers for a contact |easily find an alternate number to contact someone
+
+|`* * *` | user | undo/redo my last action | recover from mistakes
 
 |`* * *` |user |add a new person |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |use acronyms instead of the full command |use the app more efficiently
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
-|=======================================================================
+|`* * *` |user |view all entries in alphabetical order |easily find the contact I'm looking for
+
+|`* * *` |user |store birthdays of my contacts |remember them
+
+|`* * *` |user |import contacts from Google Contacts/iCloud |populate the app without having to add manually
+
+|`* * *` |user |email a contact by clicking his/her email or shortcut |talk to people faster
+
+|`* * *` |user |view the home address of a contact in Google Maps |get directions quicker
+
+|`* * *` |experienced user |remove duplicate contacts |have a cleaner Address Book
+
+|`* * *` |user |view my favourite contacts immediately |save time by not having to find them
```

Comment:
If you want this, perhaps you need a function to be able to flag contacts as 'favourite'?